pkg/conf/clusters.go

```diff
@@ -18,6 +18,12 @@ type Clusters struct {
 // fnv1a hasing following by the jump one. Every record is sent to only one host.
 const JumpCluster string = "jump"
 
+// LB cluster type distributes incoming datapoints using xxHash modulo N, where
```

Comment:
please, revise this comment. 
do we use xxHash?

Reply:
Updated.